### ISSUE TITLE
GATK4: status of some Picard/GATK3.7 tools missing from alpha?

### ISSUE THREAD

**Issue by sooheelee** [broadinstitute/gatk#3084]
Yes, what are the plans? I too would like to know.

---
What are the plans for the tools available in Picard 2.9.2 or GATK3.7 that are not in GATK4 alpha? Is the plan eventually to port everything to GATK4? Or are some being permanently sent out to pasture?

I have specifically noticed as missing:
  - CollectVariantCallingMetrics
  - SetNmMdAndUqTags
  - the -gvcf option for ValidateVariants

This Issue was generated from your [forums] 
[forums]: http://gatkforums.broadinstitute.org/gatk/discussion/9736/gatk4-status-of-some-picard-gatk3-7-tools-missing-from-alpha/p1

**Comment by droazen** [comment]
All Picard tools will be packaged in the final GATK4 release.

Selected additional tools from GATK3, such as `DepthOfCoverage` and `VariantAnnotator`, will also be ported.

**Comment by droazen** [comment]
Closing as answered

**Comment by sooheelee** [comment]
Does final release = beta release or the production release?

**Comment by droazen** [comment]
The official 4.0 release, not the beta release.

**Comment by sooheelee** [comment]
Thank you.

**Comment by magicDGS** [comment]
I would like to eventually port the indel realignment pipeline, but I don't know if I will have time. @sooheelee, maybe you can tell me if people is interested on it? I think that it is still important for Pool-Seq data, where haplotype-based calling is not usually performed.

**Comment by vdauwera** [comment]
@magicDGS I've heard anecdotal comments from people saying they would like to continue using indel realignment for non-standard use cases, and it certainly makes sense for some of those.

**Comment by sooheelee** [comment]
Yes, @magicDGS I have answered a continuous stream of questions on indel realignment in the forum and these questions are ongoing. Here's one I answered just last week: http://gatkforums.broadinstitute.org/gatk/discussion/comment/39359#Comment_39359. So it appears that it is a step that folks continue to use. 

I would add that indel realignment is especially important for Mutect1 pipelines, which continues as the standard in somatic calling, pending performance of Mutect2. Even if Mutect2 were to outperform Mutect1 calling, because of the extreme difference in compute between the two, I conjecture folks will continue to use Mutect1. In fact, the only sane way to output a BAM for the entirety of a somatic sample set is via indel realignment. Somatic folks are keen on manual review and indel realignment is the cheapest way to get there.
